documentation/js-sdk-aml-guide/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,96 @@
+////
+Copyright (c) 2022 Robert Bosch Manufacturing Solutions GmbH
+
+See the AUTHORS file(s) distributed with this work for additional
+information regarding authorship.
+
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+SPDX-License-Identifier: MPL-2.0
+////
+
+= Introduction
+
+The Aspect Model loader can be used to load an Aspect Model to traverse and access the different information a model provides (units, descriptions, constraints, etc.) to implement generic frontends capable to automatically adapt to the underling Aspect Model.
+For example a new added property and the related value could be shown in the UI including all information (name, description and unit) without changing any line of code.
+
+== Getting Started
+
+=== Install from the public npm (Currently not available):
+
+[source,bash]
+----
+npm install @sds/aspect-model-loader
+----
+
+=== Install from github release repository:
+
+[source,bash]
+----
+npm install https://github.com/OpenManufacturingPlatform/sds-sdk-js-aspect-model-loader/releases/download/<TAG_VERSION>/sds-aspect-model-loader-x.x.x.tgz
+----
+
+== Usage
+
+This package contains two methods for loading ttl files(string value of the .ttl files).
```

Comment:
IMHO we should rephrase this sentence a bit to make clear that a ttl file refers to a Turtle file. Proposal: `This package contains two methods for loading Turtle (.ttl) files (string value of the .ttl files).`